runtime/oti/j9nonbuilder.h

```diff
@@ -1415,13 +1437,16 @@ typedef struct J9SharedClassConfig {
 	UDATA  ( *isBCIEnabled)(struct J9JavaVM *vm) ;
 	void  ( *freeClasspathData)(struct J9JavaVM *vm, void *cpData) ;
 	void  ( *jvmPhaseChange)(struct J9VMThread *currentThread, UDATA phase);
+	void  (*storeGCHints)(struct J9VMThread* currentThread, UDATA heapSize1, UDATA heapSize2, BOOLEAN forceReplace);
+	IDATA  (*findGCHints)(struct J9VMThread* currentThread, UDATA *heapSize1, UDATA *heapSize2);
 	struct J9MemorySegment* metadataMemorySegment;
 	struct J9Pool* classnameFilterPool;
 	U_32 softMaxBytes;
 	I_32 minAOT;
 	I_32 maxAOT;
 	I_32 minJIT;
 	I_32 maxJIT;
+	struct J9SharedLocalStartupHints* localStartupHints;
```

Comment:
The structure could be allocated/stored here, rather than using a pointer and allocating the memory at runtime. The memory isn't much, and it might be useful for debugging to have the hints stick around.

Reply:
Fixed